data/campaigns/emp03.wmf/scripting/starting_conditions.lua

```diff
@@ -67,7 +67,8 @@ hq = prefilled_buildings(p1,
          [{0,0,0,0}] = 45,
       }
    }
-)
+)[1]
```

Comment:
do we need this [1] for something here as we have the hq to address the HQ

Reply:
`prefilled_buildings` returns an array of buildings, so before this change `hq` was actually a one-element array containing the HQ building. This was never noticed because we previously did not use the `hq` variable for anything afterwards.